xml/installation-installation-ses_integration.xml

```diff
@@ -303,8 +313,10 @@ rgw keystone verify ssl = false # If keystone is using self-signed
     Enabling &rados; gateway replaces the existing &objstore; endpoint with the
     &rados; gateway endpoint.
    </para>
-   <para>
-    <emphasis role="bold">Enabling HTTPS, Creating and Importing a Certificate</emphasis>
+</section>
```

Comment:
The radosgw support in the SES integration is keyed off of the ses_config.yml containing radosgw_urls entries.  If there are no radosgw_urls, then it won't be enabled.   

```
...
radosgw_urls:
  - https://ses-osd3:8080/swift/v1
```

Reply:
I think perhaps I'm missing something here. Is that not what it indicates on lines 283 - 287?
```
   <para>
    &rados; gateway integration can be enabled (disabled) by adding (removing)
    the following line in the <filename>ses_config.yml</filename>:
   </para>
   <screen>ses_radosgw_url: "https://192.168.56.8:8080/swift/v1"</screen>
```